plugins/wasm-rust/Cargo.toml

```diff
@@ -10,3 +10,4 @@ proxy-wasm = "0.2.1"
 serde = "1.0"
 serde_json = "1.0"
 uuid = { version = "1.3.3", features = ["v4"] }
+multimap = "0"
```

Comment:
As a wrapper for the plugin, it is still necessary to consider the version number of dependencies.

Reply:
The dependency version I want to express is  `0.*`